### ISSUE TITLE
Cheatsheets now only one column instead of two

### ISSUE THREAD

**Issue by brntbeer** [github/training-kit#640]
## Overview
**TL;DR**
I noticed https://services.github.com/on-demand/downloads/subversion-migration/ is now only a single column, it used to be two? This is the same for the Git Cheatsheets as well (see the PDFs for the 2 column versions)

#### Screenshots
Current
![screen shot 2018-06-11 at 3 04 37 pm](https://user-images.githubusercontent.com/45141/41233218-0806fac0-6d89-11e8-9bc2-c06a30294ce0.png)

Original (older?)
![screen shot 2018-06-11 at 3 05 20 pm](https://user-images.githubusercontent.com/45141/41233223-0e586e0e-6d89-11e8-8e7f-d99e63bc3ce8.png)

#### Next steps
I don't know css that well, but in the web inspector i just grabbed the first column inside of `markdown-body` class and added `float: left;` and the last column to `float:right;`
- [ ] Add new CSS for `markdown-body>*:first-child` and `markdown-body>*:last-child`

cc @github/services-training: anyone have any idea on how to do this? or should I just attempt to mock something up?

**Comment by JasonEtco** [comment]
Thanks for the report @brntbeer - not sure how or when this happened, but I'm on the case 👍
